Makefile

```diff
@@ -20,3 +20,7 @@ postgres:
 pygments:
 	out=$$(edb gen-meta-grammars edgeql) && \
 		echo "$$out" > edb/edgeql/pygments/meta.py
+
+
+update:
+	pip install -Ue .[test]
```

Comment:
This isn't related but I added this because I was tired of forgetting to add `[test]` when I update the build.

Reply:
I'd also add [docs].